FWCore/Utilities/interface/IndexSet.h

```diff
@@ -0,0 +1,42 @@
+#ifndef FWCore_Utilities_IndexSet_h
+#define FWCore_Utilities_IndexSet_h
```

Comment:
Unfortunately, FWCore/Utilities is becoming the 'kitchen sink' of CMSSW, but I also do not see an obvious other candidate. If this class might ever be added to a data product, then perhaps DataFormats/Common would be better.

For now, let's leave this class here. However, please add documentation into the header.